app/scripts/controllers/registration.js

```diff
@@ -66,6 +73,17 @@ angular
       ];
     };
 
+    $scope.getValidFirstPage = registrantId =>
```

Comment:
We have to have a function to dynamically check what the first valid page is per registrant. We can't use ```validPages[0].id``` because the array of valid pages is generated from the current registrant's registrant type which could be different from the registrant they are trying to navigate to.